Vendor/iOS/MSAlertController/MSAlertController.m

```diff
@@ -146,18 +147,37 @@ + (void)presentNextAlertAnimated:(BOOL)animated {
   }
 }
 
-// FIXME: This macro is not safe for nil arguments as types in this case cannot be inferred.
+#define ARRAY_FROM_ARGS(...) ({\
+ [NSArray arrayWithObjects: !(sizeof( (char[]){#__VA_ARGS__} ) == 1) ? __VA_ARGS__ : [NSNull null], nil];\
```

Comment:
I don't think that `sizeof` will work correctly here. Also, we need some tests for it

Reply:
Updated the implementation